code/game/objects/items/devices/von-krabin.dm

```diff
@@ -0,0 +1,71 @@
+/obj/item/device/von_krabin
+	name = "Von-Krabin Stimulator"
+	desc = "The Von-Krabin Stimulator."
+	icon = 'icons/obj/faction_item.dmi'
+	icon_state = "von-krabin"
+	item_state = "von-krabin"
+	flags = CONDUCT
+	slot_flags = SLOT_BELT
+	throwforce = WEAPON_FORCE_HARMLESS
+	throw_speed = 1
+	throw_range = 5
+	price_tag = 20000
+	w_class = ITEM_SIZE_NORMAL
+	origin_tech = list(TECH_BIO = 9, TECH_MAGNET = 9)
+
+	var/active = FALSE
+	var/area_radius = 7
+
+	var/buff_power = 5
+
+	var/stats_buff = list(STAT_BIO, STAT_COG, STAT_MEC)
+	var/list/mob/living/carbon/human/currently_affected = list()
+
+/obj/item/device/von_krabin/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	check_for_faithful(list())
+	..()
+
+/obj/item/device/von_krabin/attack_self()
+	if(active)
+		active = FALSE
+		STOP_PROCESSING(SSobj, src)
+		check_for_faithful(list())
+	else
+		active = TRUE
+		START_PROCESSING(SSobj, src)
+	return
+
+/obj/item/device/von_krabin/Process()
+	..()
+	if(!active)
+		return
+	var/list/mob/living/carbon/human/affected = range(area_radius, src)
+	check_for_faithful(affected)
+	update_icon()
+
+/obj/item/device/von_krabin/proc/check_for_faithful(list/affected)
+	var/got_follower = FALSE
+	var/list/mob/living/carbon/human/no_longer_affected = currently_affected - affected
+	currently_affected -= no_longer_affected
+	for(var/mob/living/carbon/human/H in no_longer_affected)
+		for(var/stat in stats_buff)
+			H.stats.removeTempStat(stat, "von_krabin")
+			to_chat(H, SPAN_NOTICE("Your knowledge of [stat] slightly decreese."))
```

Comment:
"your knowledge of [stat] slightly decreases once you leave the von krabin's influence."

Reply:
Fixed.